### PR TITLE
Coverage controllers projects

### PR DESCRIPTION
**Pull request by olehyev138**
Improve projects controllers coverage score to 100%

### CODE REVIEW

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -309,10 +311,6 @@ def set_award
       @can_award = awardable_types_result.can_award
     end
 
-    def project_detail_path
```
Comment (aquabu) [inline app/controllers/projects_controller.rb:312]: @olehyev138 just checking why this is deleted? Is it no longer used?

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -349,25 +347,12 @@ def project_props(project)
...
     def token_props(token)
-      if token.present?
-        token.as_json(only: %i[name symbol _token_type]).merge(
-          image_url: GetImageVariantPath.call(attachment: token.logo_image, resize_to_fill: [25, 18]).path
-        )
-      end
-    end
+      return if token.nil?
 
-    def mission_props(mission)
```
Comment (aquabu) [inline app/controllers/projects_controller.rb:359]: Is this also no longer used?

**Review by aquabu — CHANGES_REQUESTED** [review]

File: app/controllers/projects_controller.rb (hunk 2 above)
Comment (olehyev138) [reply app/controllers/projects_controller.rb:359]: this is not used too

Comment (olehyev138) [reply app/controllers/projects_controller.rb:359]: @aquabu they are not used and so I removed theme instead add tests.

**Review by aquabu — APPROVED** [review]

**Pull request merged**